source/System.cpp

```diff
@@ -957,6 +987,13 @@ void System::UpdateNeighbors(const Set<System> &systems, double distance)
 	// even if it is farther away than the maximum distance.
 	for(const System *system : links)
 		neighborSet.insert(system);
+	// Even if we are not permanently connected to that system via hyperdrive,
+	// It will still be reachable by Jump Drive.
+	for(auto &link : randomLinks)
+		neighborSet.insert(link.first);
```

Comment:
Can a "random link" always be traversed by jump drive, even if the link isn't actually present?

Reply:
the random link just is a hyperdrive link that is sometimes there and sometimes not
I don't see a reason to prevent JD travel, that'd be done by another functionality

Reply:
So, why are you saying the system will still be reachable by jump drive even if hte random link isn't actually there?

Reply:
a jump drive doesnt need a hyperdrive link to reach a system? links always have been hyperdrive links

Reply:
There's still the consideration of distance, which you seem to be ignoring here.

Reply:
I don't think so? do you mean that with this you could reach an out of reach system by jumping? I'll test it